Libs/MRML/DisplayableManager/vtkMRMLInteractionWidgetRepresentation.cxx

```diff
@@ -1141,13 +1147,20 @@ void vtkMRMLInteractionWidgetRepresentation::UpdateTranslationHandleOrientation(
   }
   else
   {
-    this->Renderer->GetActiveCamera()->GetDirectionOfProjection(viewDirection_World);
-    this->Renderer->GetActiveCamera()->GetViewUp(viewUp_World);
+    camera->GetViewUp(viewUp_World);
   }
-  vtkTransform::SafeDownCast(this->Pipeline->HandleToWorldTransform->GetInverse())->TransformVector(viewDirection_World, viewDirection_Handle);
+  worldToHandleTransform->TransformVector(viewUp_World, viewUp_Handle);
 
   for (int i = 0; i < 3; ++i)
   {
+    if (!this->GetSliceNode())
```

Comment:
Could you add a comment describing which case we are handling here ?

Reply:
In retrospect looking at the code, it seemed like a case of premature optimization.
I removed the distinction between slice node and 3D/VR views for this case as GetHandleToCameraVectorWorld can handle all types of views.